affiliates/argentina.swagger.json

```diff
@@ -451,6 +451,10 @@
             "product" : {
               "type" : "string",
               "example" : "PAYDAY"
+            },
+            "provider_redirect_url" : {
```

Comment:
this is not how it works, we have `_links` object between it

Reply:
Added _links object in between